test/attr.test.js

```diff
@@ -55,11 +55,11 @@ describe('#attr', () => {
     it('fails when the actual does not match the expected', (wrapper) => {
       expect(() => {
         expect(wrapper).to.have.attr('id', 'invalid')
-      }).to.throw(`to have a 'id' attr`)
+      }).to.throw("to have a 'id' attr")
```

Comment:
Maybe consider migrating to eslint? It's a lot more flexible and it has IDE integrations.